<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MExportArchive</class>
 <widget class="QDialog" name="MExportArchive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>export archive</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/books.png</normaloff>:/new/icons/icons/books.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblText">
     <property name="text">
      <string>Choose a target file and parts of the archive for backup.
(This function can be used for data backup or for the data transfer between different versions of Marcion.)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btFile">
       <property name="text">
        <string>choose file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbCompress">
       <property name="text">
        <string>compress after export
(bzip2)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbWork">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>works (required)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCat">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>categories (recommended)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAuth">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>authors (recommended)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbData">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>stored data
(individual files and TGZs, it may take some time)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbIndex">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>indexes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtFile</tabstop>
  <tabstop>btFile</tabstop>
  <tabstop>cbCompress</tabstop>
  <tabstop>cbWork</tabstop>
  <tabstop>cbCat</tabstop>
  <tabstop>cbAuth</tabstop>
  <tabstop>cbData</tabstop>
  <tabstop>cbIndex</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MExportArchive</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MExportArchive</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
